### ISSUE TITLE
Warning: log() expects parameter 1 to be float, string given

### ISSUE THREAD

**Issue by adamwinn** [peppeocchi/php-cron-scheduler#30]
I was following this example
```
$scheduler->php('script.php')->then(function ($output) {
    log('Job executed!');
}, true);
```
and I got `Warning: log() expects parameter 1 to be float, string given`

**Comment by peppeocchi** [comment]
@adamwinn sorry for the confusion but that example is not a working example, it's there just to show that you can do whatever you want with the output of your script after is being executed.

The `log` function should be your own implementation of a logging system.

I use [Seldaek/monolog](https://github.com/Seldaek/monolog) in almost every project, it's easy to use and a simple implementation will look like 
```php
<?php

require_once __DIR__.'/vendor/autoload.php';

use GO\Scheduler;

use Monolog\Logger;
use Monolog\Handler\StreamHandler;

$log = new Logger('scheduler-logger');
$log->pushHandler(new StreamHandler('path/to/your.log', Logger::WARNING));

$scheduler = new Scheduler();

$scheduler->php('script.php')->then(function ($output) use ($log) {
    $log->info('Job executed!');
}, true);

$scheduler->run();
```

#### TL;DR
V1 of this Scheduler has a built in functionality for logging, in V2 that was removed, I added that example to provide an alternative on how to implement your custom log.
